charts/4allportal/values.yaml

```diff
@@ -243,6 +243,13 @@ webdav:
     repository: httpd
     tag: 2.4.57
 
+  events:
+    enabled: false
+    image:
```

Comment:
A value 'events.mountPrefix' needs to be added to set a prefix for relativizing $path and $dest variables of the log.
See other review part for details.

Reply:
Done.